### PR TITLE
Removed excess whitespace from image, removed en-us culture info from…

### PR DESCRIPTION
**Pull request by jennifermarsman**
Removed excess whitespace from image, removed en-us culture info from a few links, and added an FAQ with info on semantic search and this accelerator

## Purpose
Removed excess whitespace from image, removed en-us culture info from a few links, and added an FAQ with info on semantic search and this accelerator

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,14 +88,27 @@ As part of the ARM template, a UI is deployed to `https://{prefix}-ui-{randomStr
...
 ___
 
 The UI is a React-based Web App available in the `SearchUI` folder. Navigate to [**SearchUI/README.md**](SearchUI/README.md) for more details on the web app. 
 
+## Frequently Asked Questions (FAQ)
+**How is this solution utilizing QnaMaker different from the [new semantic search functionality](https://docs.microsoft.com/azure/search/semantic-search-overview) in Cognitive Search?**
 
-## Disclaimer
+The semantic search question answering does have a functional overlap with this solution accelerator using QnA Maker.  Here's what you need to know.  
++ The QnA extracted by QnA Maker in this solution accelerator are high precision. Also, these QnAs can be edited in the qnamaker.ai portal.
```
Comment (dereklegenzoff) [inline README.md:100]: How about something like "The QnA extracted by QnA Maker in this solution accelerator are high precision. This means that this type of QnA is valuable for scenarios where there's a set of questions users frequently ask and you want to be able to provide a precise answer. These QnAs can also be edited in the qnamaker.ai portal giving you full control over the answers."

File: README.md [hunk 2]
```diff
@@ -88,14 +88,27 @@ As part of the ARM template, a UI is deployed to `https://{prefix}-ui-{randomStr
...
 
 The UI is a React-based Web App available in the `SearchUI` folder. Navigate to [**SearchUI/README.md**](SearchUI/README.md) for more details on the web app. 
 
+## Frequently Asked Questions (FAQ)
+**How is this solution utilizing QnaMaker different from the [new semantic search functionality](https://docs.microsoft.com/azure/search/semantic-search-overview) in Cognitive Search?**
 
-## Disclaimer
+The semantic search question answering does have a functional overlap with this solution accelerator using QnA Maker.  Here's what you need to know.  
++ The QnA extracted by QnA Maker in this solution accelerator are high precision. Also, these QnAs can be edited in the qnamaker.ai portal. 
++ The QnAs provided by the semantic search feature are high recall.  These are extracted at runtime and non-customizable.
```
Comment (dereklegenzoff) [inline README.md:101]: How about something like "The QnAs provided by the semantic search feature are high recall. This gives you the ability to get meaningful answers to much more specific questions. These are extracted at runtime and non-customizable."

**Review by dereklegenzoff — COMMENTED** [review]
Thanks for adding this in and for fixing the links!

File: README.md (hunk 1 above)
Comment (jennifermarsman) [reply README.md]: Great suggestion, thanks Derek!  Added in commit da8dd04b21de120299b9d2aae76ed4a1438912d5.

File: README.md (hunk 2 above)
Comment (jennifermarsman) [reply README.md]: Added in commit da8dd04b21de120299b9d2aae76ed4a1438912d5.

**Pull request merged**